airbyte-cdk/java/airbyte-cdk/db-sources/src/main/java/io/airbyte/cdk/integrations/debezium/internals/DebeziumPropertiesManager.java

```diff
@@ -76,15 +77,40 @@ public Properties getDebeziumProperties(
     props.setProperty("max.queue.size.in.bytes", BYTE_VALUE_256_MB);
 
     // WARNING : Never change the value of this otherwise all the connectors would start syncing from
-    // scratch
-    props.setProperty(TOPIC_PREFIX_KEY, getName(config));
+    // scratch.
```

Comment:
Are there any potential re-sync issues with existing syncs? I assume that because in these cases syncs are already in a broken state it shouldn't be an issue, but wanted to confirm

Reply:
Yeah that's what I'm thinking too. The reformatting code is just borrowed from debezium sanitize code so I would expect them to recognize invalid name in the same way.